cluster/jbuild

```diff
@@ -0,0 +1,40 @@
+(* -*- tuareg -*- *)
+#require "unix"
+
+let flags = function
+| [] -> ""
+| pkgs ->
+  let cmd = "ocamlfind ocamlc -verbose" ^ (
+    List.fold_left (fun acc pkg -> acc ^ " -package " ^ pkg) "" pkgs
+  ) in
+  let ic = Unix.open_process_in
+    (cmd ^ " | grep -oEe '-ppx? (\"([^\"\\]|\\.)+\"|\\w+)'")
+  in
+  let rec go ic acc =
+    try go ic (acc ^ " " ^ input_line ic) with End_of_file -> close_in ic; acc
+  in
+  go ic ""
+
+let rewriters_ppx = ["ppx_deriving_rpc"; "ppx_sexp_conv"]
+let rewriters_camlp4 = ["rpclib.idl -syntax camlp4o"]
+
+let coverage_rewriter = ""
+(* (preprocess (pps)) doesn't work with camlp4 and  the other ppx derivers,
+ it complains about missing rpc_of_t *)
+let rewriters_ppx =
+  let is_coverage = try Unix.getenv "BISECT_ENABLE" = "YES" with Not_found -> false in
+  if is_coverage then "bisect_ppx" :: rewriters_ppx else rewriters_ppx
+
+
+let () = Printf.ksprintf Jbuild_plugin.V1.send {|
+(jbuild_version 1)
+
+(library
+ ((name xcp_cluster)
+  (public_name xcp.cluster)
+  (modules (:standard))
```

Comment:
I think you should do `(modules (:standard \ cluster_client))` and completely delete `cluster_idl`

Reply:
Sorry, an oversight. I meant to add a `cluster_cli` as with the other components. Then remove that from the `modules` list, and add the executable rules